### ISSUE TITLE
Portal  Link to NCBI Data

### ISSUE THREAD

**Issue by ssarrafan** [microbiomedata/nmdc-server#370]
Provide hyperlink to same biosample/SRA record for metagenomes/metatranscriptomes

Priority - High
Urgency - High

**Comment by jbeezley** [comment]
We likely need this information added to the schema and the workflow pipelines. Once that is done, it's a quick task to add it to the UI.

**Comment by kfagnan** [comment]
Similar comments for ESS-DIVE, IMG, and NCBI - need to understand scope and where the information will come from.

**Comment by aclum** [comment]
biosample alternative identifiers are are in biosample metadata, the berkeley schema refactor will have sra identifiers for data objects but raw data is not current shown on the data portal. If we work on this we should consider breaking this into two parts. One ticket for biosample which we are in a better position to implement and one for linking to sra data.

**Comment by aclum** [comment]
NCBI biosample links are available for samples.

**Comment by aclum** [comment]
First need to expose the data generation identifiers and then show the experimental identifiers. Low priority, users can navigate to sra identifiers via biosample identifiers.

**Comment by ssarrafan** [comment]
@aclum is this still something you'd want to get done for the renewal?

**Comment by aclum** [comment]
no
